rai/lib/utility.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <rai/lib/utility.hpp>
+
+void release_assert_internal (bool check, const char * check_expr, const char * file, unsigned int line)
+{
+	if (check)
+	{
+		return;
+	}
+
+	std::cerr << "Assertion (" << check_expr << ") failed " << file << ":" << line << std::endl;
+	abort();
```

Comment:
Will calling `abort()` cause problems for our test suite?

Reply:
As in, will it abort the test suite instead of just failing the test

Reply:
This is what `assert(3)` does, so our test suite should handle it.  I did test it and this is the result:

```
[ RUN      ] block.send_serialize_json
Assertion (false) failed /home/rkeene/devel/raiblocks/rai/lib/numbers.cpp:299
Aborted
```

Reply:
Ok, if its behavior matches assert, then LGTM.